src/utils/getArticleContents.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+
+const fluid = require("infusion");
+const { JSDOM } = require("jsdom");
```

Comment:
I'm just noticing that this project already has linkedom. We've switched to that in other projects so might as well just use that here instead and remove jsdom.